Brokkr/src/main/cpp/commands/IntakeSpin.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "commands/IntakeSpin.h"
+
+IntakeSpin::IntakeSpin(Claw& mClaw)
```

Comment:
Name parameters with lower case. Variables prefixed with `m` are reserved for member variables.

```
IntakeSpin::IntakeSpin(Claw& claw)
: mClaw(claw) 
```